SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+Security alerts related to the hosting and delivery of the ecips.ethereumclassic.org web site will be addressed and will be taken by either the ecip editors or org admins. Security concerns with I individual ecips should be addressed in the `discussion to` locations.
+Security alerts will not be made publicly until mitigating solutions are identified. Once a remedy is found, dependably will automate the repair process.
+
+This site is free and open source in accordance with the site license which can be found at LICENSE.md or COPY.md
+ 
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| master  | :white_check_mark: |
+
+
+## Reporting a Vulnerability
+
+Please contact security@ethereumclassic.org with any website or ecip related security concerns.
```

Comment:
This section should include all editors and org admins' PGP keys. Here's mine: https://github.com/sorpaas.gpg

Reply:
I can link to this repo https://github.com/ethereumclassic/volunteer/tree/master/Volunteer-Public-Keys